build

```diff
@@ -97,6 +97,7 @@ if [ "$resolve_cname" = 1 ]; then
 fi
 
 make_opts=(
+	CI="$CI"
```

Comment:
Not sure if `$CI` is the most intuitive control var name.... maybe make it more explicit what this dose, along the lines of `NO_PRINT_TIMESTAMPS` or something 🤔